workers/batch-job-templates/tximport.tpl.json

```diff
@@ -0,0 +1,72 @@
+{
+    "jobDefinitionName": "${{USER}}_${{STAGE}}_TXIMPORT",
+    "type": "container",
+    "parameters": {
+        "job_name": "",
+        "job_id": ""
+    },
+    "containerProperties": {
+        "image": "${{DOCKERHUB_REPO}}/${{SALMON_DOCKER_IMAGE}}",
+        "vcpus": 1,
+        "memory": 32768,
```

Comment:
are these supposed to be hardcoded ram values?

Reply:
For some of them yes, for others we template in a variety of RAM tiers. I just double checked all the values that are hardcoded and they seem to match the values we used for the Nomad jobs of the same types.